### PR TITLE
feat: add uploadpurging config

### PR DESCRIPTION
**Pull request by zyyw**
Closes: https://github.com/goharbor/harbor-helm/issues/1118

Signed-off-by: Shengwen Yu <yshengwen@vmware.com>

### CODE REVIEW

**Comment by reasonerjt** [comment]
I think you also need to update the README.md?

File: values.yaml [hunk 1]
```diff
@@ -913,3 +913,11 @@ trace:
     insecure: true
     timeout: 10s
 
+# enable purge _upload directories
```
[the remark this answers is not in the document]
Comment (zyyw) [reply values.yaml]: good idea

**Comment by zyyw** [comment]
@reasonerjt the PR has been updated per your comments on it.

**Review by stonezdj — APPROVED** [review]
LGTM

**Review by reasonerjt — APPROVED** [review]

**Pull request merged**